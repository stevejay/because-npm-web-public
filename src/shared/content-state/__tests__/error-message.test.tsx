import React from "react";
import { render } from "react-testing-library";
import ErrorMessage from "../error-message";

test("displays 404 error correctly", async () => {
  const { container } = render(<ErrorMessage error={404} />);
  expect(container).toMatchSnapshot();
});

test("displays 500 error correctly", async () => {
  const { container } = render(<ErrorMessage error={500} />);
  expect(container).toMatchSnapshot();
});
