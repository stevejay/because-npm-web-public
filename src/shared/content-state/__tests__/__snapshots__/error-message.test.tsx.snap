// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays 404 error correctly 1`] = `
<div>
  <section
    class="container"
  >
    <div
      class="container"
    >
      <span
        class="errorCode"
      >
        404
      </span>
      <h2
        class="heading title"
      >
        Not Found
      </h2>
    </div>
  </section>
</div>
`;

exports[`displays 500 error correctly 1`] = `
<div>
  <section
    class="container"
  >
    <div
      class="container"
    >
      <span
        class="errorCode"
      >
        500
      </span>
      <h2
        class="heading title"
      >
        Server Error
      </h2>
    </div>
  </section>
</div>
`;
